### ISSUE TITLE
Missing code highlighting

### ISSUE THREAD

**Issue by carstendietrich** [i-love-flamingo/docs.flamingo.me#4]
Currently code blocks arent highlighted.. Update mkdocs config to make it work: https://squidfunk.github.io/mkdocs-material/reference/code-blocks/

**Comment by bassamadnan** [comment]
can  i take this up? will highlight using ```go wherever i can (or for other parts)

edit: pymdownx.superfences extension in mkdocs does allow you to specify line numbers as well. I think i'll add this too, since in documentation line numbers are referenced.

**Comment by tessig** [comment]
@bassamadnan feel free, this is very appreciated

**Comment by bassamadnan** [comment]
thanks! Just to be sure, changes would be made in the core repositry itself right? Since currently the build process relies on the documentation over there. Can I create an issue, ask to be assigned  and submit a PR over there?

**Comment by tessig** [comment]
yes, you can directly open a PR in the other repos and reference this issue in the description. There is no need for extra issues

**Comment by bassamadnan** [comment]
Sounds good, i also noticed that the Makrdowns already contain ```go/bash wherever necessary. I see no reason why the highlight wouldn't appear in the snippets, which has left me confused.

**Comment by tessig** [comment]
I see, since this issue is quite old, I guess the only thing left is updating the configuration in this repo then

**Comment by IvanMaidurov** [comment]
Sorry for the very long response, thank you very much for the pull request, left only one small comment there.

**Comment by bassamadnan** [comment]
all good! glad i was of help :)

**Comment by IvanMaidurov** [comment]
Code highlighting is there and search also works, @carstendietrich thanks for raising this issue and @bassamadnan thanks for solving it.
